### PR TITLE
add follow support, release v1.1

### PR DESCRIPTION
**Pull request by snarfed**

### CODE REVIEW

**Comment by snarfed** [comment]
I've been poking at https://app.travis-ci.com/dashboard a bunch of ways but haven't managed to get it to run on this PR. @pfefferle any ideas?

**Comment by snarfed** [comment]
I'm actually thinking of just removing Travis altogether, since there are no unit tests, so it's not doing anything right now.

**Comment by pfefferle** [comment]
Oh yes, please remove it. I replaced Travis with GitHub Actions everywhere I could, since Travis changed their handling of Open Source projects.

File: indieweb-press-this.php [hunk 1]
```diff
@@ -49,8 +49,8 @@ public static function bookmarklets_page() {
 			One big <a href="https://indieweb.org/">IndieWeb</a> <em>raison d’être</em> is using your own
 			web site to <a href="https://indieweb.org/reply">reply</a>,
 			<a href="https://indieweb.org/like">like</a>, <a href="https://indieweb.org/repost">repost</a>,
-			and <a href="https://indieweb.org/rsvp">RSVP</a> to posts and events. You do this by
-			annotating links on your site with simple
+            <a href="https://indieweb.org/follow">follow</a>, and <a href="https://indieweb.org/rsvp">RSVP</a>
```
Comment (pfefferle) [inline indieweb-press-this.php:52]: There seems to be something different with the indents

**Review by pfefferle — APPROVED** [review]
LGTM

File: indieweb-press-this.php (hunk 1 above)
Comment (snarfed) [reply indieweb-press-this.php:52]: tabs vs spaces, whee 😆

**Pull request merged**